src/components/SystemProfileHealthCard/index.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * TODO: This component is not complete. It was prototyped as part of
+ * https://jiraent.cms.gov/browse/EASI-1368, but has not undergone any 508 testing,
+ * UX review, etc.
+ */
+
+import React from 'react';
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  Grid,
+  GridContainer
+} from '@trussworks/react-uswds';
+import classnames from 'classnames';
+
+import './index.scss';
+
+type SystemProfileHealthCardProps = {
+  heading: string;
+  body: React.ReactNode | string;
+  footer: React.ReactNode | string;
+  status: 'success' | 'warning' | 'fail';
```

Comment:
I'm using a similar type for my work on EASI-1367, so it might be worth putting in `src/types` or `src/constants/enums`.

Reply:
6efe49bc